sparse/tests/test_core.py

```diff
@@ -201,6 +208,23 @@ def test_elemwise_binary_empty():
         assert z.data.shape == (0,)
 
 
+def test_elemwise_binary_OR():
+    size = 3
+    dimension = 3
+    shape = ((size,) * dimension)
+
+    x = np.zeros(shape=shape,dtype=bool)
+    xx = COO.from_numpy(x)
+
+    y = np.zeros(shape=shape,dtype=bool)
+    y[0,1,2] = True
```

Comment:
It might be wise to test numeric values as well.  The `|` and `&` operators in NumPy perform bitwise operations.